store/etcd/etcd.go

```diff
@@ -495,7 +495,7 @@ func (l *etcdLock) Lock(stopChan chan struct{}) (<-chan struct{}, error) {
 func (l *etcdLock) holdLock(key string, lockHeld chan struct{}, stopLocking <-chan struct{}) {
 	defer close(lockHeld)
 
-	update := time.NewTicker(time.Duration((l.ttl / 3) + 1))
+	update := time.NewTicker((time.Duration(l.ttl) * time.Second / 3) + 1)
```

Comment:
We can remove the `+ 1` too, this is useless as we add `1ns`. Also adding `1s` will fail because if we specify a `LockTTL` of `1s` it will be `1,33333..s` and we will constantly lose the lock.